templates/themes/categories/frames.html

```diff
@@ -5,6 +5,7 @@
         <meta name="viewport" content="width=device-width,initial-scale=1.0">
 
     <link rel="stylesheet" media="screen" href="{{ config.url_stylesheet }}">
+    <link rel="stylesheet" media="screen" href="stylessheets/dark_red.css">
```

Comment:
wait why don't we just have this in our config then? It seems to me that instead of changing this, we can just put ```stylessheets/dark_red.css``` as the ```url_stylesheet``` key of instance_config.php.

The same goes for the news template.

Also you misspelled "stylesheets" and I'm not sure how this ever worked on the dev server now.